### DISCUSSION TITLE
How to set other laser excitation?

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#8] [General]
In Raman we are taking four elaser to calculate Raman shift. Is it possible to take other values of elaser to get spectra? How to get full phonon dispersion using QERaman?
![Uploading 17061803059796763385803009922400.jpg…]()

**Comment by nguyen-group** [comment]
Hi,
> Is it possible to take other values of elaser to get spectra?

You can change the laser energy in the input file, see: https://github.com/nguyen-group/QERaman/wiki/Input-variables-description

> How to get full phonon dispersion using QERaman?

You can not get the full phonon dispersion using QERaman, only the phonon at Gamman point is used in QERaman.
For the full phonon, you can see at: https://github.com/nguyen-group/QE-SSP/tree/master/gr/phonon

Best,
Nguyen
